packages/oui-numeric/src/numeric.controller.js

```diff
@@ -72,6 +63,18 @@ export default class {
         this.previousValue = this.model;
     }
 
+    $postLink () {
+        // Sometimes the digest cycle is done before dom manipulation,
```

Comment:
This shouldn't be specified everywhere, maybe have a document to explain the design choices